temoa-uiuc/data_files/10_uiuc.sql

```diff
@@ -742,28 +742,28 @@ INSERT INTO "Efficiency" VALUES('uiuc', 'E85', 'E85VCL', 2028, 'UVCL', 0.734,'1
-INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2028, 'UVCL', 0.031,'1.0 kWh = 0.031 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2029, 'UVCL', 0.031,'1.0 kWh = 0.031 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2030, 'UVCL', 0.031,'1.0 kWh = 0.031 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2021, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2022, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2023, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2024, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2025, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2026, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2027, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2028, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2029, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'ELCVCL', 2030, 'UVCL', 0.11,'1.0 kWh = 0.11 gal gsl');
 
 INSERT INTO "Efficiency" VALUES('uiuc', 'ethos', 'IMPH2', 2021, 'H2', 1.00,'pure hydrogen import');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2021, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2022, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2023, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2024, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2025, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2026, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2027, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2028, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2029, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
-INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2030, 'UVCL', 1.00,'1 kg h2 = 1.0 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2021, 'UVCL', 2.40,'1 kg h2 = 2.4 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2022, 'UVCL', 2.40,'1 kg h2 = 2.4 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2023, 'UVCL', 2.40,'1 kg h2 = 2.4 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2024, 'UVCL', 2.40,'1 kg h2 = 2.4 gal gsl');
+INSERT INTO "Efficiency" VALUES('uiuc', 'H2', 'H2VCL', 2025, 'UVCL', 2.40,'1 kg h2 = 2.4 gal gsl');
```

Comment:
We've discussed this before, but just to be clear: The energy content of 1kg H2 = 1 gallon gsl but the conversion from potential to kinetic energy is more efficient for H2 vehicles than gasoline vehicles, right?

Reply:
Correct. Another way to see it is using the mileage per gallon. 1 gallon of gasoline (in a conventional vehicle) yields X miles, and 1 kg of hydrogen (in an FCEV) yields Y miles. Y being greater than X (I am using X and Y cause I don't remember the numbers from the top of my head, but you get the idea).